kolibri/auth/models.py

```diff
@@ -26,120 +38,258 @@ class KolibriValidationError(KolibriError):
     pass
 
 
-class BaseUser(AbstractBaseUser):
+##############################################
+# MORANGO: These pieces will live in Morango, but dummies/stubs are temporarily included here until Morango is ready.
+##############################################
```

Comment:
Again, the stuff in this MORANGO interlude are just stubs, and no need to review them. They will be more fully featured and documented within Morango itself, later.